cypress/e2e/selecting.cy.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+
+describe('react-drag-to-select example', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('can select some items', () => {
+    cy.get('.container')
+      .trigger('mousedown', 10, 10, {
+        eventConstructor: 'MouseEvent',
+        button: 0,
+      })
+      .trigger('mousemove', 400, 150, {
+        eventConstructor: 'MouseEvent',
+        button: 1,
+      })
+      .trigger('mouseup', 400, 150, {
+        eventConstructor: 'MouseEvent',
+        button: 1,
```

Comment:
Why do we have `button: 1` here? Also, do we even need this options - isn't `trigger('mousedown', 10, 10)` enough?